### DISCUSSION TITLE
`core::error::Error` support?

### DISCUSSION THREAD

**Discussion by rinarakaki** [zkat/miette#251] [Q&A] [answered]
I searched Issues and PRs and couldn't find a related one. Are there ongoing efforts or plans to migrate to `core::error::Error`? Thanks.

**Comment by zkat** [comment, marked as answer]
What's the purpose? `no_std` support? There's currently no plans to have miette work in `no_std` scenarios, and moving to `core::error::Error` would be a breaking change, so idk.
